Makefile

```diff
@@ -6,11 +6,12 @@ BUILD_DIR        = "${GOPATH}/src/github.com/openshift/ansible-service-broker/bu
 PREFIX           ?= /usr/local
 BROKER_CONFIG    ?= $(PWD)/etc/generated_local_development.yaml
 
+vendor:
```

Comment:
HAH. This was me putting build first so I could lazily run `make` to rebuild. Then you educated me on DEFAULT_GOAL, sensei.